frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -32,15 +30,87 @@ const CampCreationPage = (): React.ReactElement => {
     campDetailsDummyOne && campDetailsDummyTwo && campDetailsDummyThree !== "";
   const isScheduleSessionsFilled =
     scheduleSessionsDummyOne && scheduleSessionsDummyTwo;
-  const isRegistrationFormFilled =
-    registrationFormDummyOne && registrationFormDummyTwo;
-
+  const isRegistrationFormFilled = visitedRegistrationPage;
   // State of what page to display.
   const [currentPage, setCurrentPage] = useState<CampCreationPages>(
     CampCreationPages.CampCreationDetailsPage,
   );
   /* eslint-enable */
```

Comment:
Since this is the part in the code where we store lots of state (sometimes long, complex lines of state), I noticed that the linter usually makes it really difficult to read, but I can enable the linter again if you suggest.